src/main-process/integration.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
I added this because I notices that it is missing. I think we want all our code to be in strict mode.